test/Test/Biscotti/CookieTest.purs

```diff
@@ -107,6 +107,35 @@ testSuite = do
         Cookie.parse "key=val" `shouldEqual` Right expected
 
     suite "parseMany" do
+      test "parse value wrapped in DQUOTEs" do
```

Comment:
Just a minor nitpick for clarity's sake.

```suggestion
      test "parse value wrapped in double-quotes" do
```

Reply:
Ha that's fine. I thought `DQUOTE` might be more "official" since it was original from RFC6265 syntax specification but I like this too.